client/my-sites/stats/activity-log/index.jsx

```diff
@@ -193,14 +218,16 @@ class ActivityLog extends Component {
 			startDate,
 		} = this.props;
 
-		const startOfMonth = moment( startDate ).startOf( 'month' ),
-			startOfMonthMs = startOfMonth.valueOf(),
-			endOfMonthMs = moment( startDate ).endOf( 'month' ).valueOf();
-		const filteredLogs = filter( logs, obj => startOfMonthMs <= obj.ts_site && obj.ts_site <= endOfMonthMs );
-		const logsGroupedByDate = map(
+		const siteOffset = this.getSiteOffsetFunc();
+
+		const startOfMonthMs = siteOffset( moment( startDate ).startOf( 'month' ) ).valueOf();
+		const endOfMonthMs = siteOffset( moment( startDate ).endOf( 'month' ) ).valueOf();
```

Comment:
> we are applying offsets to both the month start/end and the timestamps

Heh, actually, we are not applying the offsets to `ts_utc` for the month filter currently, so it is not working properly. I'll fix it to ignore offsets for this op.